### ISSUE TITLE
v0.36.0 breaks on function calls

### ISSUE THREAD

**Issue by lencioni** [brigade/scss-lint#414]
Running scss-lint on the following file:

```scss
.my-selector {
  width: my-function(2);
}
```

fails in 0.36.0 with:

```
SCSSLint::Linter::PropertyUnits raised unexpected error linting file test.scss: 'undefined method `value' for my-function(2):Sass::Script::Tree::Funcall'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/linter/property_units.rb:21:in `visit_prop'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/sass-3.4.13/lib/sass/tree/visitors/base.rb:36:in `visit'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/linter.rb:127:in `visit'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/linter.rb:138:in `block in visit_children'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/linter.rb:136:in `each'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/linter.rb:136:in `visit_children'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/sass-3.4.13/lib/sass/tree/visitors/base.rb:38:in `visit'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/linter.rb:127:in `visit'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/linter.rb:138:in `block in visit_children'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/linter.rb:136:in `each'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/linter.rb:136:in `visit_children'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/sass-3.4.13/lib/sass/tree/visitors/base.rb:36:in `block in visit'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/linter/property_units.rb:10:in `visit_root'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/sass-3.4.13/lib/sass/tree/visitors/base.rb:36:in `visit'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/linter.rb:127:in `visit'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/linter.rb:21:in `run'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/runner.rb:52:in `run_linter'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/runner.rb:33:in `block in find_lints'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/runner.rb:31:in `each'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/runner.rb:31:in `find_lints'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/runner.rb:17:in `block in run'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/runner.rb:16:in `each'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/runner.rb:16:in `run'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/cli.rb:54:in `scan_for_lints'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/cli.rb:48:in `act_on_options'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/lib/scss_lint/cli.rb:27:in `run'
/Users/joe/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/scss-lint-0.36.0/bin/scss-lint:6:in `<top (required)>'
/Users/joe/.rbenv/versions/2.2.1/bin/scss-lint:23:in `load'
/Users/joe/.rbenv/versions/2.2.1/bin/scss-lint:23:in `<main>'
Report this bug at https://github.com/brigade/scss-lint/issues
```

Besides exclusions, here is the config:

```yaml
linters:
  ColorVariable:
    enabled: false
  NameFormat:
    convention: BEM
  SelectorFormat:
    convention: hyphenated_BEM
  UrlFormat:
    enabled: false
```

Disabling `PropertyUnits` for now to avoid this.

**Comment by sds** [comment]
Thanks for the report! Fixed in eb78c5633fb9719130a9a7437d028c0e4f8ed88f.
